classes/submission/sectionEditor/SectionEditorSubmission.inc.php

```diff
@@ -707,16 +707,19 @@ function getHighlightClass() {
 							strtotime($reviewAssignment->getDateNotified()) : 0;
 						$dateConfirmed = $reviewAssignment->getDateConfirmed() ?
 							strtotime($reviewAssignment->getDateConfirmed()) : 0;
+						$dateDue = $reviewAssignment->getDateDue() ?
```

Comment:
@marchitelli, this isn't related to the rest of this PR, I don't think; could you send it separately?